fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -1235,6 +1236,10 @@ public ParseNode visitSimpleFunctionCall(StarRocksParser.SimpleFunctionCallConte
             Expr e1 = (Expr) visit(context.expression(0));
             Expr e2 = (Expr) visit(context.expression(1));
             if (!(e2 instanceof IntervalLiteral)) {
+                if (functionName.toUpperCase().equals(FunctionSet.DATE_FLOOR.toUpperCase())) {
```

Comment:
This check should do in FunctionAnalyzer.


all function names are lowercase，needn't toUpperCase.

Reply:
moved to ExpressionAnalyzer